src/ui/utils/telemetry.ts

```diff
@@ -14,7 +14,6 @@ export function setupTelemetry() {
     "Current thread has changed",
     "Failed to load Stripe.js",
     "Stripe.js not available",
-    "Received status code 500",
```

Comment:
So are we still going to send all Apollo 500 errors to sentry with the same subject line? I was picturing us stopping these here, but then doing something like:

```
Sentry.captureException(new Error(graphQLErrors.join(": "));
```

in the Apollo error link.

Reply:
I considered this but didn't want to intro customized error handling. So we still `console.error` these errors but with a more descriptive log and sentry will still capture them as before.